.github/workflows/publish_report_aave.yml

```diff
@@ -0,0 +1,39 @@
+name: publish-book-aave
```

Comment:
We must no change the contents of the plutocracy report book, published at https://butterymoney.github.io/gov_analysis/report/plutocracy_report.html

We must keep this report final, as it's already been published together with a Mirror article.

So, I suggest deleting the `publish_report.yml` file or commenting-out its contents.